source/comply/legal-hold.rst

```diff
@@ -43,86 +43,65 @@ Establish a policy for when to implement a legal hold. This should be developed
-
-1. In Mattermost, go to **System Console > Compliance > Compliance Export**.
-2. Set **Enable Compliance Exports** to **true**.
-3. Set the **Compliance Export time**. Choose a time when fewer people are using your system, such as 2:00AM your local time.
-4. Set the export file format to **Actiance XML**.
-5. Learn more about compliance export configuration options in our :doc:`product documentation </comply/compliance-export>`.
+When the Legal Hold integration is enabled, you can configure when it runs in the format ``HH:MM ±HHMM`` and ``+0000`` for UTC, and can optionally configure a custom Amazon S3 bucket for legal holds by specifying Amazon S3 configuration settings. Alternatively, you can use existing :ref:`Amazon S3 file storage <configure/environment-configuration-settings:amazon s3 bucket>` for legal holds, where access keys are supported when using IAM roles.
 
 .. note::
 
-    - For legal hold, only XML format is supported at this time. 
-    - Support for additional file formats, such as CSV or EML, are under consideration.
-
-3 - Leverage compliance API in your DLP tool of choice
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Only Amazon S3 is supported for Mattermost legal holds at this time, for increased reliability, compliance, and automation capabilities. Support for additional file storage options such as MinIO or local file storage are under consideration. Learn more about file storage configuration options in our :ref:`product documentation <configure/environment-configuration-settings:file storage>`.
 
-Connect it directly to the Amazon S3 bucket where compliance exports are stored. Our recommended DLP tool of choice is `Smarsh <https://www.smarsh.com/>`_. Learn more about their legal hold & e-discovery processes in their `documentation <https://www.smarsh.com/platform/enterprise/discovery>`_.
-
-You may optionally also use other tools that connect with Amazon S3, such as Active360, AwareHQ, Onna, or Trellix, though our team will be able to provide limited support using these tools.
-
-4 - (Optional) Configure a data retention policy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-This is typically implemented alongside legal hold, though is not required.
+(Optional) Configure a data retention policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. In Mattermost, go to **System Console > Compliance > Data Retention Policies**.
-2. Set either a **Global retention policy** that applies to all messages and files older in your system, or a **custom retention policy** for specific teams or channels.
-3. Learn more about data retention configuration options in our :doc:`product documentation </comply/data-retention-policy>`.
+You can optionally configure a :doc:`data retention policy </comply/data-retention-policy>` for the legal hold data you want to retain and delete to align with compliance requirements.
```

Comment:
This wording suggests the retention policy affects how long legal hold data is kept. That is not the case.  Retention policy only affects data in the MM database and file attachments.  Legal hold data is kept until the legal hold is released.